### PR TITLE
Updated Readme File

### PR DESCRIPTION
**Pull request by GouraV-Chatterjee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 The 100 Days of Code-Python Challenge is a personal commitment to engage intensive Python programming focused on equipping you with advanced Python skills, best coding practices, and hands-on experience. It covers a wide range of topics, from basic Python concepts to advanced topics and real-world projects.
 
-## Why am I taking this challenge?
+## Who can take this challenge?
```
Comment (sree-hari-s) [inline README.md:5]: also need to change the content below to match the title

Comment (GouraV-Chatterjee) [reply README.md:5]: @sree-hari-s Updated the content too...

File: README.md [hunk 2]
```diff
@@ -2,12 +2,12 @@
...
```
Comment (sree-hari-s) [inline README.md:6]: Don't repeat the same in every point make it professional

File: README.md [hunk 3]
```diff
@@ -2,12 +2,12 @@
...
 
-- To become a proficient Python developer.
-- To stay updated with the latest Python trends and technologies.
-- To build a strong foundation for a successful career in programming.
-- To join a community of Python enthusiasts and professionals.
+- Individuals who want to become a proficient Python developer.
```
Comment (sree-hari-s) [inline README.md:7]: Don't repeat the same in every point make it professional

Comment (GouraV-Chatterjee) [reply README.md]: Removed the repetative part and paraphrased the lines!.

**Review by sree-hari-s — APPROVED** [review]

**Comment by sree-hari-s** [comment]
LGTM, Please put a ⭐ on this project, Try to contribute more. 😁

**Pull request merged**